go/internal/tinder/driver_mocked.go

```diff
@@ -82,6 +82,46 @@ func (s *MockDriverServer) FindPeers(ns string, limit int) (<-chan p2p_peer.Addr
 	return ch, nil
 }
 
+func (s *MockDriverServer) FindPeersAsync(ctx context.Context, outChan chan<- p2p_peer.AddrInfo, ns string, limit int) {
```

Comment:
FindPeers() and FindPeersAsync() have some same code, could you refractor it?

Reply:
I could but I don't find that more readable.
That would require to create a :
`FindPeersAsync(context.Context, chan<- p2p_peer.AddrInfo, string, ...p2p_discovery.Option) (<-chan p2p_peer.AddrInfo, error)` and this method would check the params chan agaisnt `nil` to know if it have to instanciate a chan, close it, ...